tensornetwork/backends/jax/jax_backend.py

```diff
@@ -16,11 +16,15 @@
 from tensornetwork.backends import base_backend
 from tensornetwork.backends.numpy import decompositions
 import numpy as np
+from tensornetwork.backends.jax import jitted_functions
+from functools import partial
 
 Tensor = Any
 
-
 # pylint: disable=abstract-method
+# pylint: disable=attribute-defined-outside-init
```

Comment:
Why was this added?

Reply:
removed it (not that `_jaxlan` is still defined outside init)